app/models/share/facebook.rb

```diff
@@ -2,24 +2,7 @@ class Share::Facebook < ActiveRecord::Base
   include Share::Variant
 
   validates :description, :title, presence: true
+  belongs_to :image
```

Comment:
it's just a practicality of foreign keys. whatever has the `belongs_to` has the table column for `image_id` or whatever key into the other table. the Image gets associated with so many things that it would be a crazy amount of key columns on the image table. this does to me open the question of whether it makes sense to associate all these things to the image model, or if it would be better to just add paperclip images to each model.